src/device-registry/models/backups/Event-29-08-2025.js

```
@@ -0,0 +1,2893 @@
+    thenCategory: AQI_CATEGORIES.hazardous,
+    thenColorName: AQI_COLOR_NAMES.hazardous,
+  },
+];
+
+// Helper function to build MongoDB $switch expressions
+// from the shared AQI branches using a specific "then" property
+const buildSwitchExpression = (thenProperty) => ({
+  $switch: {
+    branches: AQI_BRANCHES.map((branch) => ({
+      case: branch.case,
+      then: branch[thenProperty],
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)